common/api/presentation-components.api.md

```diff
@@ -531,7 +532,7 @@ export function PresentationTreeNodeRenderer(props: PresentationTreeNodeRenderer
 // @alpha (undocumented)
 export interface PresentationTreeNodeRendererProps extends TreeNodeRendererProps {
     // (undocumented)
-    nodeRenderer?: (props: TreeNodeRendererProps) => React.ReactNode;
+    onFilter?: (node: TreeModelNode) => void;
```

Comment:
Maybe pass `TreeNodeItem` instead of `TreeModelNode` to be consistent with `TreeEvents`